src/python/CRABClient/Commands/SubCommand.py

```diff
@@ -384,11 +384,17 @@ def __init__(self, logger, cmdargs=None, disable_interspersed_args=False):
         if self.cmdconf['requiresRucio']:
             if os.environ.get('RUCIO_HOME', None):
                 username = getUsername(self.proxyfilename, logger=self.logger)
-                from rucio.client import Client
-                os.environ['RUCIO_ACCOUNT'] = username
-                self.rucio = Client()
-                me = self.rucio.whoami()
-                self.logger.info('Rucio client intialized for account %s' % me['account'])
+                try:
+                    from rucio.client import Client
+                    from rucio.common.exception import RucioException
+                    os.environ['RUCIO_ACCOUNT'] = username
+                    self.rucio = Client()
+                    me = self.rucio.whoami()
+                    self.logger.info('Rucio client intialized for account %s' % me['account'])
+                except RucioException as ex:
```

Comment:
what happens is that exception is not catched and crab command fails miserably and user will send us the traceback so we can try to figure out how is it possible that the rucio import fails given that we source rucio code in crab starting script.
I can put the imports before the try-except, but IIUC it will be same behavior.

Reply:
@mapellidario let me know if you that answes satisfactorily your question and if yes and you have nothing more, please approve